content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,197 @@
+/>
+
+[Docs](https://docs.sourcegraph.com/admin/updates#multi-version-upgrades)
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### View usage and measure the value of our platform with new and enhanced in-product analytics
+Our new analytics, introduced with version 3.42, make it easier for admins to understand user engagement, measure efficiency in terms of time saved, and help calculate the value of utilizing Sourcegraph and its features. In-product analytics now include ROI-based metrics such as projected value and dev-time-saved for all Sourcegraph features, and individual user usage. Admins can access the analytics, configure the date range from one week to three months, and export the details. 
+
+![In-product anlaytics](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Search.png "In-product analytics")
+
+[Docs](https://docs.sourcegraph.com/admin/analytics)
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Uncover developer time saved using Browser and IDE extensions 
+Admins can now understand the adoption and amount of time saved from using IDE and Browser extensions across your organization. Additionally, to make it faster and easier to roll out our VS Code extension organization wide, we've created an example Batch Change that adds the extension to your repository’s recommended extensions. Once added, users will be recommended the Sourcegraph extension when opening the repository if they don't have it installed already. We’ve also updated our instructions on how to automatically install our Browser extensions for users in your Google Workspace organization. 
+
+![Browser and IDE extensions analytics](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/4.0/Browser.png "Browser and IDE extensions analytics")
+
+[Docs](https://docs.sourcegraph.com/admin/analytics)
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Easily export traces using OpenTelemetry
+
+[OpenTelemetry (OTEL)](https://opentelemetry.io/) is an open specification for exporting observability data into a user’s preferred systems. We aim to make OpenTelmetry the default way to quickly and easily export observability data from Sourcgraph, starting with tracing in Sourcegraph 4.0. This is a breaking change since we now require customers to export tracing using the [OpenTelemetry collector](https://opentelemetry.io/docs/collector/), which will now be deployed by default in all Sourcegraph deployment methods, replacing the Jaeger agents and collectors we used to ship by default.
+
+[Docs](https://docs.sourcegraph.com/admin/observability/opentelemetry)
+
+<Badge text="Admin" color="violet" size="small" onClick={() => window.open("URL_GOES_HERE", "_self")} />
+
+#### Quickly see the status on your repository and permissions syncing
```

Comment:
@lcolston08 - this update is a bit repetitive/vague. Instead of saying "we added many quality of life improvements," we should share what we specifically improved.

Reply:
@ryphil Could you please provide more details on what exactly we updated?